docs/getting-started.md

```diff
@@ -138,3 +133,41 @@ provide your own certificates, please refer to [Securing Control Plane](/docs/se
 
 To use antctl, the Antrea command-line tool, please refer to this
 [guide](/docs/antctl.md).
+
+## Features
+
+### Antrea Network Policy
+Besides Kubernetes NetworkPolicy, Antrea also implements its own Network Policy
+CRDs, which provides advanced features including: policy priority, tiering, deny
+action, external entity, and policy statistics. For more information on usage of
+Antrea Network Policies, refer to the [Antrea Network Policy document](antrea-network-policy.md).
+
+### IPsec encryption
+Antrea supports encrypting GRE tunnel traffic with IPsec. To deploy Antrea with
+IPsec encryption enabled, please refer to [this guide](/docs/ipsec-tunnel.md).
+
+### Network flow visibility
+Antrea supports exporting network flow information using IPFix, and provides
+reference cookbook on how to visualized the exported network flows using Elastic
+Stack and Kibana dashboards. For more information, refer to the [network flow
+visibility document](network-flow-visibility.md).
+
+### Octant UI
+Antrea builds a UI plugin in Octant which can show runtime information of Antrea
+components and perform Antrea Traceflow operations. Refer to [this guide](octant-plugin-installation.md)
+to learn how to install Octant and the Antrea plugin.
+
+### OVS hardware offload
+Antrea can offload OVS flow processing to the NICs that support OVS hardware
+offload. The hardware offload can improve OVS performance significantly. For
```

Comment:
maybe clarify here: to the NICs that support OVS kernel hardware offload using TC

since there are 2 different ways of offloading OVS flows (TC / DPDK)

Reply:
Done